### PR TITLE
Fixes memory leak in traffic_crashlog

### PR DESCRIPTION
**Pull request by randall**
Closes issue #5708

### CODE REVIEW

**Comment by randall** [comment]
[approve ci autest]

**Comment by randall** [comment]
[approve ci autest]

**Comment by randall** [comment]
[approve ci autest]

**Comment by randall** [comment]
[approve ci autest]

File: src/traffic_crashlog/traffic_crashlog.cc [hunk 1]
```diff
@@ -262,6 +262,8 @@ main(int /* argc ATS_UNUSED */, const char **argv)
 
   Error("wrote crash log to %s", logname);
 
+  TSfree(logname);
```
Comment (masaori335) [inline src/traffic_crashlog/traffic_crashlog.cc]: It looks like `logname` should be also freed before returning 1 in line 231.

Comment (maskit) [inline src/traffic_crashlog/traffic_crashlog.cc]: Wow, this code uses the both `TSfree` and `ats_free`.

Comment (zwoop) [reply src/traffic_crashlog/traffic_crashlog.cc]: Yeh, I don't think we should TSfree() here, only ats_free(), right ? TSfree() ought to be only used in plugins.

Comment (randall) [reply src/traffic_crashlog/traffic_crashlog.cc]: Updated

**Pull request merged**